Packs/Code42/Integrations/Code42/Code42.yml

```diff
@@ -624,66 +649,6 @@ script:
     - contextPath: Code42.User.UserID
       description: The ID of a Code42 User.
       type: String
-  - arguments:
```

Comment:
This appears to be removing a bunch of stuff re: legal hold commands, can you sync up with master?